src/Components/ArtworkFilter/ArtworkFilters/__tests__/Utils.tsx

```diff
@@ -0,0 +1,149 @@
+import {
+  RenderOptions,
+  render as originalRender,
+  screen,
+} from "@testing-library/react"
+import {
+  ArtworkFilterContextProps,
+  ArtworkFilterContextProvider,
+  useArtworkFilterContext,
+} from "Components/ArtworkFilter/ArtworkFilterContext"
+import { ReactElement } from "react"
+
+/**
+ * A test helper that can be used to render components that depend
+ * on an enclosing `ArtworkFilterContext`.
```

Comment:
Tried to get some good intellisense going…

<img width="1446" alt="Screenshot 2023-10-20 at 11 28 31 AM" src="https://github.com/artsy/force/assets/140521/a51d1527-46ec-4cd6-9b69-5bcf26612312">